bundles/org.openhab.binding.gpio/README.md

```diff
@@ -39,71 +39,204 @@ ExecStart=/usr/bin/pigpiod
 
-### GPIO digital input channel
+### GPIO pigpio-digital-input channel configuration
+
+Input channels provide a read-only value of the gpio pin state using the `OnOffType` datatype.
 
-Set the number of the pin in `gpioId`.
-If you want to invert the value, set `invert` to true.
-To prevent incorrect change events, you can adjust the `debouncingTime`.
-Using `pullupdown` you can enable pull up or pull down resistor (OFF = Off, DOWN = Pull Down, UP = Pull Up).
+GPIO Pin:
 
-### GPIO digital output channel
+The gpio pin number on the Pi that the channel will monitor.
 
-Set the number of the pin in `gpioId`.
-If you want to invert the value, set `invert` to true.
+Invert:
 
-## Full Example
+Inverts the value of the gpio pin before reflecting the value on the channel. Useful
+for active low gpio pins where you want the channel state to reflect an ON value when
+the pin is low (OFF).
+
+Delay Time:
+
+Sets a delay value in milliseconds that the gpio pin must remain at prior to updating the channel state. 
+This is the same as switch debouncing or hysteresis. Default value is 10 milliseconds. Increase this value
+for noisy inputs.
+
+Pull Up/Down Resistor:
+
+Sets the mode of operation for the internal pull up/ down resistor on the gpio pin.
+Set this to OFF if you use external pull up/down resistors.
+
+Edge Detection Mode:
+
+Sets the mode of operation for the pin edge detection mode. If you are not sure what
+the use case is for this, leave at the default value of `Either Edge`. This is the most common
+mode that gpio inputs are used.
+
+### GPIO pigpio-digital-output channel configuration
+
+Output channels provide a means of controlling the output value of the gpio pin using the `OnOffType` datatype.
+
+GPIO Pin:
+
+The gpio pin number on the Pi that the channel will control.
+
+Invert:
+
+Inverts the value of the channel state before commanding the gpio pin. Useful to 
+simulate active low gpio pins.
+
+Pulse:
+
+Time in milliseconds that must elapse before the Pulse Command is sent to the channel.
+Default value is 0, which disables the Pulse feature.
+
+Pulse Command:
+
+Together with the Pulse configuration, can be used to create a one shot or momentary output.
+This is useful to simulate momentary button presses or to drive motors for a predefined amount
+of time.
+
+  Off             : When the ON command is issued to the channel. The Pulse feature will send an
+                    OFF command after the Pulse duration.
+  On              : When the OFF command is issued to the channel. The Pulse feature will send an
+                    ON command after the Pulse duration.
+  Blink           : Cycles the channel ON, OFF, ON indefinitely with a 50% duty cycle. The Blink
+                    operation continues regardless of the commanded channel state. This was originaly
+                    developed as a way to flash a status LED to visually confirm that a remote pigpiod
+                    instance has connectivity to openHAB.
```

Comment:
This will be rendered without any formatting, i.e. shown just as a long paragraph. Perhaps it could be put into a table, or at least a bullet list?